src/db/spanner/support.rs

```diff
@@ -188,7 +189,13 @@ impl StreamedResultSet {
     ///
     /// Returns false when the stream is finished
     fn consume_next(&mut self) -> Result<bool> {
-        let mut partial_rs = if let Some(result) = self.stream.next() {
+        let (result, stream) = System::new("").block_on(
```

Comment:
I'm pretty sure the overhead of calling System::new("") on every single bsos request is going to be untenable.

Reply:
I changed it so that there is a threadlocal System for every thread in the db pool.